File: curso python/ejemplos python/problema18.py
#dado 3 numeros devolver los numeros en orden ascendente
a = int (input('num1: '))
b= int (input ('num2: '))
c= int (input ('num3: '))
if a>b:
    if a>c:
        mayor = a
    else:
        mayor =c
else:
    if b>c:
        mayor=b
    else:
        mayor = c
if a<b and a<c:
    menor =a
elif b<a and b<c:
    menor =b
else:
    menor =c
intermedio= (a+b+c)-(mayor + menor)

print ('menor: ',menor)
print (f'intermedio: {intermedio}')
print ('mayor: ',mayor)

File: curso c++/ejemplos/problema18.cpp
//dado 3 numeros devolver los numeros en orden ascendente
#include <iostream>
using namespace std;
int main (){
    int a,b,c,menor,intermedio,mayor;
    cout << "introduce el numero: ";
    cin >> a;
    cout << "introduce el numero: ";
    cin >> b;
    cout << "introduce el numero: ";
    cin >> c;
    if (a>b && a>c)
        mayor =a;
    else
        if (b>a && b>c)
            mayor =b;
        else
            mayor =c;

    if (a<b && a<c)
        menor = a;
    else
        if (b<a && b<c)
            menor =b;
        else
            menor =c;
    intermedio = (a + b + c)-(mayor + menor);
    cout << "menor: "<<menor<<endl;
    cout <<"intermedio: " <<intermedio<< endl;
    cout << "mayor: "<<mayor<< endl;

    return 0;
}